extension/bootstrap.js

```diff
@@ -232,7 +275,33 @@ const windowListener = {
 
 this.install = function(data, reason) {};
 
-this.startup = function(data, reason) {
+this.startup = async function(data, reason) {
+  studyUtils.setup({
+    studyName: config.study.studyName,
+    endings: config.study.endings,
+    addon: { id: data.id, version: data.version },
+    telemetry: config.study.telemetry,
+  });
+  studyUtils.setLoggingLevel(config.log.studyUtils.level);
+  const variation = await chooseVariation();
+  studyUtils.setVariation(variation);
+
+  // TODO Import config.modules?
+
+  if ((REASONS[reason]) === "ADDON_INSTALL") {
```

Comment:
Per discussion with @mythmon, it's better to use the other way in order to use constants instead of just arbitrary string values. This can also help with linting.